index.html

```diff
@@ -6,7 +6,11 @@
 		<script src="http://code.jquery.com/jquery-3.2.1.min.js"></script>
 		<script src="static/index.js"></script>
 		<link rel="stylesheet" href="static/index.css">
-		<meta name="viewport" content="width=800">
+		<meta name="viewport" content="width=700">
+		<!--Dont' know whats nessacary in the code underneath this-->
+		<link rel="icon" 
```

Comment:
This code is only necessary if you want to control the favicon from your own repository - there's a favicon.ico anyways, so it would show without this, anyways. You could choose one or the other.

Reply:
Oh ok we can remove it then.